R/agglomerate.R

```diff
@@ -49,6 +49,9 @@
 #'   experiments be removed prior to agglomeration? This prevents to many
 #'   nested alternative experiments by default (default:
 #'   \code{strip_altexp = TRUE})
+#'   
+#' @param other_label A single \code{character} valued used as the label for the
+#'   summary of non-prevalent taxa. (default: \code{other_label = "Other"})
```

Comment:
This could be added to getPrevalence.R above agglomerateByPrevalence function if possible.